packages/playwright-test/src/loader.ts

```diff
@@ -143,7 +142,19 @@ export class Loader {
     this._fullConfig.shard = takeFirst(config.shard, baseFullConfig.shard);
     this._fullConfig._ignoreSnapshots = takeFirst(config.ignoreSnapshots, baseFullConfig._ignoreSnapshots);
     this._fullConfig.updateSnapshots = takeFirst(config.updateSnapshots, baseFullConfig.updateSnapshots);
-    this._fullConfig.workers = takeFirst(config.workers, baseFullConfig.workers);
+
+    const workers = takeFirst(config.workers, '50%');
```

Comment:
I set the `50%` on this line instead of `baseFullConfig`, to don't mess up with `FullConfigInternal.workers` typing for no reason. I believe it is not useful to have `FullConfigInternal.workers` as `number|string`  since it actually represent the number of workers internally.

What do you think ?

Reply:
Yep, this is good.